crates/corro-types/src/broadcast.rs

```diff
@@ -80,6 +89,7 @@ pub enum BroadcastV1 {
 pub enum ChangeSource {
     Broadcast,
     Sync,
+    PriorityBroadcast,
```

Comment:
I don't think we ever match on this, so I'd be in favor of not having it.

Reply:
Oh yeah, good catch. I thought I had already removed it